modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -1148,14 +1148,57 @@ func (suite *KeeperTestSuite) TestQueryUnreceivedPackets() {
 		{
 			"invalid seq",
 			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				suite.coordinator.Setup(path)
+
 				req = &types.QueryUnreceivedPacketsRequest{
-					PortId:                    "test-port-id",
-					ChannelId:                 "test-channel-id",
+					PortId:                    path.EndpointA.ChannelConfig.PortID,
+					ChannelId:                 path.EndpointA.ChannelID,
```

Comment:
Why do you change this lines? The original code is not changed. 
https://github.com/cosmos/ibc-go/pull/3346/files#diff-d66fe0702464ada7d4549d10b4e8b562fc101673cb5f60319ab8419b63fd0dfdR1154-R1155